pkg/process/monitor/process_monitor.go

```diff
@@ -346,15 +338,14 @@ func (pm *ProcessMonitor) Subscribe(callback *ProcessCallback) (UnSubscribe func
 
 func (pm *ProcessMonitor) Stop() {
 	pm.m.Lock()
-	defer pm.m.Unlock()
 	pm.refcount--
```

Comment:
I think that we should not decrement the refcount if it is negative already

Reply:
:+1: